client/blocks/domain-to-plan-nudge/index.jsx

```diff
@@ -0,0 +1,333 @@
+/**
+ * External dependencies
+ */
+import debugFactory from 'debug';
+import React, { PropTypes, Component } from 'react';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+import { get } from 'lodash';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import DismissibleCard from 'blocks/dismissible-card';
+import { recordTracksEvent } from 'state/analytics/actions';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getSite, isCurrentSitePlan } from 'state/sites/selectors';
+import { plansList, PLAN_FREE, PLAN_PERSONAL } from 'lib/plans/constants';
+import { storedCardPayment } from 'lib/store-transactions';
+import { getStoredCards } from 'state/stored-cards/selectors';
+import QueryStoredCards from 'components/data/query-stored-cards';
+import { emptyCart } from 'lib/cart-values';
+import { add } from 'lib/cart-values/cart-items';
```

Comment:
Can we please `{ add as addItemToCartFactory }` ? The naming and usage of this function is terrible when imported